api/src/certification/session-management/infrastructure/repositories/invigilator-access-repository.js

```diff
@@ -5,13 +5,13 @@ const create = async function ({ sessionId, userId }) {
   await knexConn('supervisor-accesses').insert({ sessionId, userId });
 };
 
-const isUserSupervisorForSession = async function ({ sessionId, userId }) {
+const isUserInvigilatorForSession = async function ({ sessionId, userId }) {
   const knexConn = DomainTransaction.getConnection();
   const result = await knexConn.select(1).from('supervisor-accesses').where({ sessionId, userId }).first();
```

Comment:
Il va falloir prévoir un travail de migrations de la BDD


```suggestion
  const result = await knexConn.select(1).from('invigilator-accesses').where({ sessionId, userId }).first();
```